docs/tutorial.rst

```diff
@@ -1264,6 +1264,16 @@ or by the index in ``$arg<n>`` environment variables.
     >>> aliases['piu'] = 'pip install -U @($args)'
     >>> aliases['cdls'] = 'cd $arg0 && ls'
 
+.. warning:: That means, if you need `@()` in your alias, you can't rely on
+             automatic argument handling anymore. Any alias involving it needs
+             to add `` $args`` at the end manually if you don't want your alias
```

Comment:
```suggestion
             to add ``$args`` at the end manually if you don't want your alias
```

Reply:
Here, the space was intentional, because shell rules and such. But I guess nobody would actually do `aliases["cmd"] = "echo$args"`.